packages/sanes-chrome-extension/src/routes/login/index.dom.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { Store } from 'redux';
+import { RootState } from '../../store/reducers';
+import { aNewStore } from '../../store';
+import TestUtils from 'react-dom/test-utils';
+import { travelToLogin } from './test/travelToLogin';
+import { whenOnNavigatedToRoute } from '../../utils/test/navigation';
+import { WELCOME_ROUTE } from '../paths';
+
+describe('DOM > Feature > Login', () => {
+  let store: Store<RootState>;
+
+  beforeEach(() => {
+    store = aNewStore();
+  });
+
+  it(`should two inputs and one button`, async () => {
+    const LoginDom = await travelToLogin(store);
+
+    const inputs = TestUtils.scryRenderedDOMComponentsWithTag(
+      LoginDom,
+      'input'
+    );
+
+    expect(inputs.length).toBe(2);
+
+    TestUtils.findRenderedDOMComponentWithTag(LoginDom, 'button');
+
+    //Bottom links
+    const links = TestUtils.scryRenderedDOMComponentsWithTag(LoginDom, 'h6');
```

Comment:
`h6` is something really generic, so if you use it, you should do extra checks. Something like:
```
expect(links.length).toBe(3);
expect(links[0].innerHTML).toBe('whatever')
...
```